project/.github/ISSUE_TEMPLATE.md

```diff
@@ -12,21 +12,20 @@
 
 #### Sonata packages
 
-```bash
-$ composer show sonata-project/*
-# Put the result here.
+```
+$ composer outdated --all 'sonata-project/*'
+# Put the result here if there is one, otherwise, just remove this paragraph.
 ```
 
 #### Symfony packages
 
-```bash
-$ composer show symfony/*
-# Put the result here.
+```
+$ composer outdated --all 'symfony/*'
 ```
 
 #### PHP version
 
-```bash
```

Comment:
Ok since your sample: https://github.com/sonata-project/SonataAdminBundle/issues/4267

Can you please do the same for composer outputs in this case?